src/Components/DogCardTemplate/DogCardTemplate.js

```diff
@@ -0,0 +1,32 @@
+import { useTranslation } from "react-i18next";
+import "./DogCardTemplate.styles.scss";
+
+const CardTemplate = ({ combinedDogData }) => {
+  const { t } = useTranslation();
+  return (
+    <div className="cardGrid">
+      {combinedDogData.map((value, index) => (
```

Comment:
you can deconstruct that value to 
```
const { name, nameDoge, traiability } = value
```
and then later just pass
```
<h5 className="card-title">{nameDog}</h5>
```

Reply:
Thanks for the tip, I will try to remember this for the future it was useful